database/migrations/2020_10_13_183125_create_post_photos_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreatePostPhotosTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('post_photos', function (Blueprint $table) {
```

Comment:
suggestion: Rename table to post_photo

Reply:
OR posts_photos?

Reply:
https://laravel.com/docs/8.x/eloquent-relationships#many-to-many

Laravel example
``` 
users
    id - integer
    name - string

roles
    id - integer
    name - string

role_user
    user_id - integer
    role_id - integer
``` 

So it should be photo_post (table name should be in  alphabetical order)